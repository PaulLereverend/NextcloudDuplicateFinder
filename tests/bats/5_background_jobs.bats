#!/usr/bin/env bats
TESTSUITE="duplicatefinder"
load helper.sh

setup() {
  load ${BATS_TEST_DIRNAME}/setup.sh
  ./occ duplicates:clear -f
}

@test "[$TESTSUITE] Test background jobs" {
  dbQuery "update oc_jobs set last_run=0,last_checked=0,reserved_at=0  where class like '%DuplicateFinder%';"
  php ./cron.php
  run ./occ -v duplicates:list
  [ "$status" -eq 0 ]

  expectedHash="37df47434c76bd4387c975aba58906ea1c0e86be6e8e7d3f7ad329017f5a9187"
  evaluateHashResult "${expectedHash}" 26 "${output}"
}
