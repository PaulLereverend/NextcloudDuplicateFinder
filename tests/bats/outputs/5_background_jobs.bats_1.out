Duplicates are: 
1121cfccd5913f0a63fec40a6ffd44ea64f9dc135c66634ba001d10bcf4302a2(file_hash)
     /admin/files/tests/3_0.txt
     /admin/files/tests/3_1.txt
     /admin/files/tests/3_2.txt
     /admin/files/tests/3_3.txt
     /tuser/files/tests/3_0.txt
     /tuser/files/tests/3_1.txt
     /tuser/files/tests/3_2.txt
     /tuser/files/tests/3_3.txt
     /tuser/files/tests2/3_0.txt
     /tuser/files/tests2/3_1.txt
     /tuser/files/tests2/3_2.txt
     /tuser/files/tests2/3_3.txt
4355a46b19d348dc2f57c046f8ef63d4538ebb936000f3c9ee954a27460dd865(file_hash)
     /admin/files/tests/1_0.txt
     /admin/files/tests/1_1.txt
     /tuser/files/tests/1_0.txt
     /tuser/files/tests/1_1.txt
     /tuser/files/tests2/1_0.txt
     /tuser/files/tests2/1_1.txt
53c234e5e8472b6ac51c1ae1cab3fe06fad053beb8ebfd8977b010655bfdd3c3(file_hash)
     /admin/files/tests/2_0.txt
     /admin/files/tests/2_1.txt
     /admin/files/tests/2_2.txt
     /tuser/files/tests/2_0.txt
     /tuser/files/tests/2_1.txt
     /tuser/files/tests/2_2.txt
     /tuser/files/tests2/2_0.txt
     /tuser/files/tests2/2_1.txt
     /tuser/files/tests2/2_2.txt
9a271f2a916b0b6ee6cecb2426f0b3206ef074578be55d9bc94f6f3fe3ab86aa(file_hash)
     /admin/files/tests/0_0.txt
     /tuser/files/tests/0_0.txt
     /tuser/files/tests2/0_0.txt
